### ISSUE TITLE
About the training data

### ISSUE THREAD

**Issue by ShangwenWang** [ICSE-2019-AUTOFIX/ICSE-2019-AUTOFIX#1]
Dear authors,

Congratulations on your great work.
I find you use the dataset from a recent OOPSLA paper for training. You introduced that you have made several preprocess to ensure the bugs are all single-statement bugs. That is really a valuable dataset. I wonder if you could release this for the convenience of future studies.

Many thanks~

**Comment by DehengYang** [comment]
Seemingly the data is available at https://github.com/OOPSLA-2019-BugDetection/OOPSLA-2019-BugDetection.

**Comment by ShangwenWang** [comment]
@DehengYang Yes, thanks. I find these two datasets possess some subtle but critical differences.

**Comment by davidswresearcher** [comment]
yes, the oopsla dataset contains all types of bugs. for this DLfix paper, we only target at the single-statement bugs, thus we processed the oopsla data to extract the single-statement bugs.
